apps/examples/mediarecorder/mediarecorder_main.cpp

```diff
@@ -107,35 +120,35 @@ class MediaRecorderTest : public MediaRecorderObserverInterface,
 	void start(const int test)
 	{
 		uint8_t volume;
-		appRunning = true;
+		mAppRunning = true;
 
-		while (appRunning) {
+		while (mAppRunning) {
 			printRecorderMenu();
 			switch (userInput(APP_ON, DELETE_FILE)) {
 			case APP_ON:
 				std::cout << "SELECTED APP ON" << std::endl;
-				mr.create();
-				mr.setObserver(shared_from_this());
+				mMr.create();
+				mMr.setObserver(shared_from_this());
 				if (test == TEST_PCM) {
 					filePath = "/tmp/record.pcm";
-					mr.setDataSource(unique_ptr<FileOutputDataSource>(
+					mMr.setDataSource(unique_ptr<FileOutputDataSource>(
```

Comment:
TEST_PCM & TEST_OPUS is redundant code, so what's going down the block?

Reply:
@JeongJunSik  you right.
fixed it.